xbmc/Application.cpp

```diff
@@ -3210,7 +3210,8 @@ PlayBackRet CApplication::PlayStack(const CFileItem& item, bool bRestart)
       }
     }
 
-    *m_itemCurrentFile = item;
+    CFileItemPtr tmp(new CFileItem(item));
```

Comment:
I stand corrected, CApplication::SetCurrentFileItem(item) would do it. Sorry for the confusion.